startup/30-user.py

```diff
@@ -583,8 +592,9 @@ def series(det='eiger4m',shutter_mode='single',expt=.1,acqp='auto',imnum=5,comme
     """
     #timing delay between calling series and start of data acquisition:
     #caput('XF:11ID-CT{ES:1}bo2',1)
-    trigger_pv='XF:11ID-CT{ES:1}bi3'
-    trigger_PV=EpicsSignal('XF:11ID-CT{ES:1}bi3',name='trigger_PV')
+    #trigger_pv='XF:11ID-CT{ES:1}bi3'
+    trigger_pv = 'XF:11ID-CT{M1}bi4'
+    trigger_PV=EpicsSignal(trigger_pv,name='trigger_PV')
```

Comment:
It is better to define ophyd objects once at the top level rather than in every call.  It is faster and improves the resource management.